specification/README.md

```diff
@@ -2852,8 +2852,9 @@ If the defined callback event has not been received during this time period, the
 
 | Parameter | Description | Type | Required |
 | --- | --- | --- | --- |
-| kind | End kind ("default", "scheduled") | enum | yes |
+| kind | End kind ("default", "scheduled", "triggered") | enum | yes |
```

Comment:
Triggered sounds to me like a manual invocation more than "scheduled"/cron :) Left a comment suggesting using "recurring" or overloading "scheduled". I think historically "cron" and "scheduled" have been used interchangeably which might make them separating here confusing. 

An example question I think users might ask - _"Scheduled doesn't mean cron in serverless workflow?"_

Reply:
Good point. Let me see if we can do the overload as you mentioned, having either interval or cron property under "scheduled" obj def. WDYT?

Reply:
fixed